clusterDeployer.py

```diff
@@ -468,27 +492,41 @@ def deploy(self) -> None:
             else:
                 logger.info("Skipping pre configuration.")
 
-            lh = host.LocalHost()
-            self.ensure_linked_to_bridge(lh)
-
-            if "masters" in self.steps:
-                self.teardown()
-                self.create_cluster()
-                self.create_masters()
-            else:
-                logger.info("Skipping master creation.")
+            if self._cc.kind != "microshift":
+                lh = host.LocalHost()
+                self.ensure_linked_to_bridge(lh)
 
-            if "workers" in self.steps:
-                if len(self._cc.workers) != 0:
-                    self.create_workers()
+                if "masters" in self.steps:
+                    self.teardown()
+                    self.create_cluster()
+                    self.create_masters()
                 else:
-                    logger.info("Skipping worker creation.")
+                    logger.info("Skipping master creation.")
+
+                if "workers" in self.steps:
+                    if len(self._cc.workers) != 0:
+                        self.create_workers()
+                    else:
+                        logger.info("Skipping worker creation.")
 
         if "post" in self.steps:
             self._postconfig()
         else:
             logger.info("Skipping post configuration.")
 
+        if self._cc.kind == "microshift":
```

Comment:
This happens post "postconfig", we need to move this code before postconfig() is called.